apps/demo-app/config/styles.json

```diff
@@ -167,7 +167,7 @@
   "menu-min-width": "rem-calc(200)",
   "menu-sub-bg": "isitlight($menu-bg, rgba($black, .05), rgba($white, .05))",
   "menu-panel-slide-speed": "$global-slide-speed",
-  "breakpoints": {
+   "breakpoints": {
```

Comment:
Please correct formatting so that there are only 2 spaced preceding the "breakpoints" attribute.